### DISCUSSION TITLE
Sponsoring / working full time

### DISCUSSION THREAD

**Discussion by Connum** [opentypejs/opentype.js#622] [Ideas]
I wanted to create this as a team discussion, but they seem to be disabled, so I'm posting this publically. @opentypejs/owners @opentypejs/maintainers 

Some time ago when the team of maintainers was expanded, including myself, it seemed as if a fresh wind was blowing and things were moving forward again, which was indeed the case for a while. But as it always is, work and life get the better of us. But my interest in maintaining this library and adding new features is still strong, and I thought about how cool it would be to actually get paid for doing this full-time (or at least for a considerable time of my work day).

I wanted to see what the team thinks about that in general and also open up the discussion for the best way to attract sponsors. Add a banner to the README? Mark the npm package as looking for funding? Replying to feature requests with the proposal for funding/sponsoring? All/a combination of that?

**Comment by fdb** [comment]
Hi, as the original author of opentype.js (and also overloaded with very different projects) I would definitely love to see the project grow and flourish. I think sponsored development could help in that. 

I think we can:
- Enable GitHub sponsorships for individual developers willing to work on the project
- Use [OpenCollective](https://opencollective.com/) to raise money for the project
- Add a page to the documentation site and a badge on the README indicating we're looking for sponsorships.

Collecting the money is just the first step, though. It would require somebody to define a roadmap with requested features. Is this also something that you're interested in @Connum ?

**Comment by Connum** [comment]
Sure, it's not just development I'm interested in. Table parsing and writing has so far been my main area of contribution, and to be honest, there are parts like font feature processing that are out of my comfort zone and which I couldn't tackle on my own anyway, so I would have to delegate that in any case, but at least I could do the prioritization and planning for that.

I think some of the first steps after securing funding at least for a short while would be to
* get the outstanding PRs merged
* go through all the issues, close what has meanwhile been implemented/fixed and make a list of missing features and existing bugs
* prioritize those bugs and features, with a rough roadmap for future versions

Then fix the bugs, implement the features, and finally strive for complete coverage of the OpenType specification.
Previous PRs have shown that some refactoring will also be necessary along the way.

So I would definitely not be able to tackle the whole thing on my own even if paid, but I feel that some kind of clean-up and clear vision would help others get involved.

**Comment by ILOVEPIE** [comment]
Sorry about not being too active, my job takes up a huge amount of my time and I was not anticipating that. I wish I could work on this more frequently. I need to find a new job.

**Reply by Connum** [reply]
> I need to find a new job.

Wouldn't it be great if our new job was this? 😉

**Comment by Connum** [comment]
See also #376
